### DISCUSSION TITLE
404 error not found

### DISCUSSION THREAD

**Discussion by Praveen03AJJARAPU** [vercel/community#3862] [Help]
### Summary

I recently deployed my code with Vercel it runs in my localhost but when deployed with Vercel it directly showing me 404 code not found.I thought it was routing but I added vercel.json it did not worked.

### Example

https://facebook-clon5.vercel.app/

### Steps to Reproduce

![Screenshot 2023-08-31 213554](https://github.com/vercel/community/assets/97183004/4116e488-6e5b-4625-a112-82b978c73ada)
![Screenshot 2023-08-31 213603](https://github.com/vercel/community/assets/97183004/60fe7a2c-089f-4207-8ca6-5642c4cef984)

**Comment by amyegan** [comment]
Hey @Praveen03AJJARAPU. If you share your public repo or a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), that will let us take a closer look at what is going wrong.

**Comment by Praveen03AJJARAPU** [comment]
https://github.com/Praveen03AJJARAPU/facebook-clone here is my repo link

**Reply by Praveen03AJJARAPU** [reply]
Anyone here

**Reply by amyegan** [reply]
It looks like your site is using Facebook's logo. While it is acceptable to deploy "clone" versions of popular sites for developer skills practice, there are a few things you need to do to make sure the deployment won't be blocked.

- Add an obvious banner at the top of the site explaining that it is a clone and not intended for real use.
- Remove any copyright or trademark infringement, including the logo, name, and prominent colors of the site you're cloning.
- Disable any login or credential-capturing mechanism to ensure visitors cannot enter their credentials for Facebook into your site.
